### DISCUSSION TITLE
Vulnarability found on path-to-regexp on latest starlite vesion

### DISCUSSION THREAD

**Discussion by sreejithkalarikkal** [withastro/starlight#2313] [Q&A] [answered]
How to fix the below vulnarability on latest version of starlite

┌─────────────────────┬────────────────────────────────────────────────────────┐
│ **high                │ path-to-regexp outputs backtracking regular            │
│                     │ expressions                                            │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Package             │ path-to-regexp                                         │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Vulnerable versions │ >=0.2.0 <8.0.0                                         │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Patched versions    │ >=8.0.0**                                                │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Paths               │ packages/healthelink-docs-prime >                      │
│                     │ @astrojs/starlight@0.27.1 > @astrojs/mdx@3.1.5 >       │
│                     │ astro@4.15.4 > path-to-regexp@6.2.2                    │
│                     │                                                        │
│                     │ packages/healthelink-docs-prime >                      │
│                     │ @astrojs/starlight@0.27.1 > astro@4.15.4 >             │
│                     │ path-to-regexp@6.2.2                                   │
│                     │                                                        │
│                     │ packages/healthelink-docs-prime >                      │
│                     │ @astrojs/starlight@0.27.1 >                            │
│                     │ astro-expressive-code@0.35.6 > astro@4.15.4 >          │
│                     │ path-to-regexp@6.2.2                                   │
│                     │                                                        │
│                     │ ... Found 11 paths, run `pnpm why path-to-regexp` for  │
│                     │ more information                                       │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-9wv6-86v2-598j      │
└─────────────────────┴────────────────────────────────────────────────────────┘
1 vulnerabilities found
Severity: 1 high

**Comment by delucis** [comment, marked as answer]
Thanks for the question @sreejithkalarikkal!

This vulnerability doesn’t really impact Starlight sites, but work is in progress in the Astro repo to fix this upstream.

You can follow [this Astro issue](https://github.com/withastro/astro/issues/11956), where I also commented explaining how this vulnerability impacts Astro (and therefore Starlight) sites.

**Comment by HiDeoo** [comment]
Closing this discussion as this issue has been [fixed](https://github.com/withastro/astro/pull/12001) upstream in the Astro repo.
